src/commands/plugins.js

```diff
@@ -1,36 +1,39 @@
-const pluginloader = require('../plugins/pluginloader');
-const utils = require('../utils');
+const pluginloader = require("../plugins/pluginloader");
+const utils = require("../utils");
 
 module.exports.register = (program) => {
-    program
```

Comment:
Hey @SalihuDickson, Thanks for sending this pull request. Will you be able to use the original code formatting? Then we can clearly see what's changed :tada: